delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -202,21 +195,18 @@ def _prepare_attributes(
             "przl": services,
```

Comment:
To move at the end of the method as some activated services should be removed (see below)

Reply:
fixed in https://github.com/OCA/delivery-carrier/pull/323/commits/f04b053b0aeca3c51756f5ad9129349ce26f4555